README.md

```diff
@@ -57,9 +59,17 @@ unless real values don't help users know what to change.
 
 ```hcl
 
+module "watsonx_assistant" {
+  source            = "terraform-ibm-modules/watsonx-assistant/ibm"
+  region            = "us-south"
+  assistant_name    = "example-wx-assistant-basic"
+  plan              = "lite"
+  resource_group_id = module.resource_group.resource_group_id
+}
+
 ```
 
-### Required access policies
+<!-- ### Required access policies -->
```

Comment:
please add the required IAM access policies

Reply:
Done